controllers/authController.ts

```diff
@@ -0,0 +1,96 @@
+import { NextFunction, Request, Response } from "express";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcryptjs";
+import { v4 as uuidv4 } from "uuid";
+import dotenv from "dotenv";
+import { PrismaClient } from "@prisma/client";
+import CreateError from "../utils/error";
+dotenv.config();
```

Comment:
this isn't necessary you did this in app.ts, once it's loaded it should be accessible through out the node app